plugins/modules/packaging/language/pear.py

```diff
@@ -166,9 +226,12 @@ def install_packages(module, state, packages):
         if state == 'latest':
             command = 'upgrade'
 
-        cmd = "%s %s %s" % (_get_pear_path(module), command, package)
-        rc, stdout, stderr = module.run_command(cmd, check_rc=False)
+        current_prompt_regex = (None, None)
+        if has_prompt and (len(prompts) > 0 and i < len(prompts)):
```

Comment:
`has_prompt == len(prompts) > 0` so:
```suggestion
        if has_prompt and i < len(prompts):
```